llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -42518,6 +42518,25 @@ bool X86TargetLowering::SimplifyDemandedVectorEltsForTargetNode(
       SDValue Insert =
           insertSubVector(UndefVec, ExtOp, 0, TLO.DAG, DL, ExtSizeInBits);
       return TLO.CombineTo(Op, Insert);
+    }
+      // Conversions.
```

Comment:
Do we need to consider other conversion nodes like `CVTP2SI/CVTP2UI/CVTSI2P/VFPROUND...`?

Reply:
Absolutely, I was thinking of adding them when I have test cases - but I can try to add them all now if you prefer?

Reply:
I'm OK with adding it later with test case. Maybe leave a comment here for now.